packages/host/app/resources/file.ts

```diff
@@ -42,7 +45,10 @@ export interface Ready {
   url: string;
   lastModified: string | undefined;
   realmURL: string;
+  size: number; // size in bytes
```

Comment:
I sometimes embed the unit in the name for codebase-wide clarity (or could it be encapsulated in a type?) `sizeInBytes`

Reply:
Good isea